content/en/data_streams/messages.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Messages
+---
+
+The Messages feature allows you to view messages at specific offsets within Kafka. Inspecting these messages can be crucial when troubleshooting issues related to specific services or data streams.
+
+## Setup
+Before diving into detailed configuration steps, here's a quick overview of what's involved in setting up the Messages feature:
+
+| Step               | Description                                                                                                                                       | Setup Guides              |
+|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------|
+| Agent setup        | An agent needs Kafka consumer integration enabled, be on at least version 7.69.0, and have remote configuration enabled                           | [Agent Setup][1]          |
+| Verify permissions | You'll need a few log access permissions which usually come standard as well as `Data Streams Monitoring Capture Messages` enabled in Datadog     | [Required Permissions][2] |
```

Comment:
can we list the permissions needed or don't talk about the log permissions?

Reply:
The `Required Permissions` link takes them to the bottom section listing out the permissions: https://docs-staging.datadoghq.com/steven.pham/dsm/steven.pham/dsm-live-messages-v2/data_streams/messages/#required-permissions

<img width="1657" height="1765" alt="Screenshot 2025-07-14 at 10 01 44 AM" src="https://github.com/user-attachments/assets/df75fb15-972d-4181-ab95-2e01c5f5332a" />